script.js

```diff
@@ -126,23 +126,12 @@ function renderQuestion() {
   buttonElems[optionBIndex].innerText = question.charDiedAfter;
 }
 
-function endGame() {
-  console.log('game ended');
-  const clear1 = document.querySelectorAll('.js-options-list');
-  clear1.forEach(elem => {
-    elem.className='is-hidden1';
-  })
-  const clear2 = document.querySelectorAll('.js-quiz-question'); 
-  clear2.forEach(elem => {
-    elem.className='is-hidden1';
-  })
-  const gametext = document.querySelectorAll('.is-hidden2'); 
-  gametext.forEach(elem => {
-    elem.className='end-text';
-  })
+function renderGameOver() {
+  document.querySelector('.js-options-list').classList.add('is-hidden');
+  document.querySelector('.js-quiz-question').classList.add('is-hidden');
+  document.querySelector('.js-game-over').classList.remove('is-hidden');
```

Comment:
to render the game over section I simply remove the `is-hidden` class from the `js-game-over` div.